physical/consul.go

```diff
@@ -75,9 +76,10 @@ func newConsulBackend(conf map[string]string) (Backend, error) {
 
 	// Setup the backend
 	c := &ConsulBackend{
-		path:   path,
-		client: client,
-		kv:     client.KV(),
+		path:       path,
+		client:     client,
+		kv:         client.KV(),
+		permitPool: NewPermitPool(64),
```

Comment:
Sure. I figured it can be made into a config parameter eventually, I just didn't want to spend the time on it right now. If you think it's important at this point, I can.